src/appetizers/Appetizers.tsx

```diff
@@ -60,7 +60,17 @@ const Appetizers = () => {
 					name,
 					price,
 				})
-				.then(() => {})
+				.then(() => {
+					axios
+						.get("http://localhost:8080/appetizers")
+						.then(response => {
+							setAppetizers(response.data);
+						})
+						.catch(errors => {
+							setError(errors.response.data.message);
+						});
+					setError("successfully added a new appetizer");
+				})
```

Comment:
a couple of things:

1. making another API call in the `.then` to refetch the data works, but the better way would be for your initial API to return the data you need. Then you set it in state directly. That makes it so you have 1 less API call.
2. you're setting the `error` state to a successful message on line 73? That seems misleading
3. If there's an error with your API on line 69, you're gonna update your state error twice.